gcp_variant_transforms/testing/data/input_files/combine

```diff
@@ -0,0 +1,2 @@
+gs://tural-test-runner/vcf_files/small/valid-4.1*
```

Comment:
Why the files are from your project? Is this file the same as `gs://gcp-variant-transforms-testfiles/small_tests/input_files/combine_input`? If yes, please use the same file name.